extensions/datastores/cassandra/src/main/java/mil/nga/giat/geowave/datastore/cassandra/util/CassandraInputFormatIteratorWrapper.java

```diff
@@ -0,0 +1,167 @@
+package mil.nga.giat.geowave.datastore.cassandra.util;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.NoSuchElementException;
+
+import org.apache.commons.lang.ArrayUtils;
+
+import mil.nga.giat.geowave.core.index.ByteArrayId;
+import mil.nga.giat.geowave.core.store.adapter.AdapterStore;
+import mil.nga.giat.geowave.core.store.base.BaseDataStore;
+import mil.nga.giat.geowave.core.store.filter.QueryFilter;
+import mil.nga.giat.geowave.core.store.index.PrimaryIndex;
+import mil.nga.giat.geowave.datastore.cassandra.CassandraRow;
+import mil.nga.giat.geowave.mapreduce.HadoopWritableSerializationTool;
+import mil.nga.giat.geowave.mapreduce.input.GeoWaveInputKey;
+
+/**
+ * This is used internally to translate Dynamo DB rows into native objects
+ * (using the appropriate data adapter). It also performs any client-side
+ * filtering. It will peek at the next entry in the HBase iterator to always
```

Comment:
this is mostly comments about DynamoDB, not Cassandra.  also peek at the next "HBase" iterator?